tests/pfcwd/test_pfcwd_function.py

```diff
@@ -277,7 +277,7 @@ def verify_pkt_cnts(self, port_type, pkt_cnt):
 
 class SetupPfcwdFunc(object):
     """ Test setup per port """
-    def setup_test_params(self, port, vlan, init=False, mmu_params=False):
+    def setup_test_params(self, port, vlan, init=False, mmu_params=False, is_dualtor=False):
```

Comment:
can check if it is dual tor in tbinfo instead of change methor definition

Reply:
Sure, update the code.